src/commands/ssh-proxy.ts

```diff
@@ -0,0 +1,101 @@
+/** Copyright © 2024-present P0 Security
+
+This file is part of @p0security/cli
+
+@p0security/cli is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, version 3 of the License.
+
+@p0security/cli is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with @p0security/cli. If not, see <https://www.gnu.org/licenses/>.
+**/
+import { authenticate } from "../drivers/auth";
+import { fsShutdownGuard } from "../drivers/firestore";
+import { sshProxy } from "../plugins/ssh";
+import { P0_PATH } from "../util";
+import { SSH_PROVIDERS, SshProxyCommandArgs } from "./shared/ssh";
+import * as fs from "fs/promises";
+import path from "path";
+import yargs from "yargs";
+
+export const sshProxyCommand = (yargs: yargs.Argv) =>
```

Comment:
how would users request sudo access? 🤔

Reply:
i just tested this and it looks like all you have to do is request sudo access for the node and when you ssh with the proxy command you'll have sudo too.

Reply:
i guess the problem here is users can't ask for sudo when they ssh, they must ask for it beforehand. 🤔